src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -192,25 +191,33 @@ public class HelixWatch3DViewModel : DefaultWatch3DViewModel
 
         private Dictionary<string, string> nodesSelected = new Dictionary<string, string>();
 
+        private Object element3DDictionaryMutex = new object();
+        /// <summary>
+        /// Mutex object
+        /// </summary>
+        public Object Element3DDictionaryMutex
+        {
+            get { return element3DDictionaryMutex; }
+        }
```

Comment:
From what I could see, the field was not being used outside of this class at all. Please, just keep the private field and delete the property. Mutex objects should not be shared publicly if possible.

Reply:
That's true, I think it was a public property before that's why I added it. But API backward compatibility broken anyway when we renamed this :) 
@mjkkirschner How do you feel like we just remove this public property before, seems no one will be using it like @mmisol  noticed.